website/docs/r/event_orchestration_router.html.markdown

```diff
@@ -72,6 +88,22 @@ The following arguments are supported:
 * `expression`- (Required) A [PCL condition](https://developer.pagerduty.com/docs/ZG9jOjM1NTE0MDc0-pcl-overview) string.
 
 ### Actions (`actions`) supports the following:
+
+#### Dynamic Routing
```

Comment:
**Annotation:** Unfortunately it seems the Terraform documentation site does **not** support deep links/anchors on H4 tags. That said I'm not immediately thinking of a better way to organize this portion of the page. Suggestions are welcome!

Reply:
I think the approach you chose is good, the Actions section is not that big so linking to Actions should be sufficient.